src/core/api_client/tests/factories/mock.factory.ts

```
@@ -18,13 +18,16 @@ export class MockFactory {
     const accessToken = token || (await TokenFactory.createToken());
     const refreshToken = await TokenFactory.createRefreshToken();
 
-    fetchMock.postOnce(tokenEndpoint, {
-      status: 200,
-      headers: { 'Content-Type': 'application/json' },
-      body: {
-        access_token: accessToken,
-        refresh_token: refreshToken,
-      },
+    fetchMock.post(tokenEndpoint, (url: string, opts: any) => {
+      this.callCount++;
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)